pkg/operator/controller/credentials-secret/controller_test.go

```diff
@@ -534,8 +584,8 @@ func testTargetSecret() *corev1.Secret {
 			Namespace: testOperandNamespace,
 		},
 		Data: map[string][]byte{
-			"key1": []byte("val1"),
-			"key2": []byte("val2"),
+			"aws_access_key_id":     []byte("val1"),
+			"aws_secret_access_key": []byte("val2"),
```

Comment:
The target secret should have `credentials` key.

Reply:
Updated